yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/yb-voyager-pg-gather-assessment-metadata.sh

```diff
@@ -48,15 +52,27 @@ if [ "$1" == "--help" ]; then
 fi
 
 # Check if all required arguments are provided
-if [ "$#" -ne 3 ]; then
-    echo "Usage: $SCRIPT_NAME <pg_connection_string> <schema_list> <assessment_metadata_dir>"
-    exit 1
+if [ "$4" != "" ]; then
+    if [ "$#" -ne 4 ]; then
+        echo "Usage: $SCRIPT_NAME <pg_connection_string> <schema_list> <assessment_metadata_dir>"
```

Comment:
In this usage, (and everwhere else even in examples..), we can add the sleep interval, right?